### PR TITLE
AUT-126 Persons endpoints

### PR DESCRIPTION
**Pull request by azayatsfv**
GET get_person_by_person_ids
GET get_user_details
POST update_persons_by_person_id

### CODE REVIEW

File: APIs/persons/client.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class PersonsClient(BaseClient):
+    CONTENT_TYPE = "Content-Type"
+    APPLICATION_JSON = "application/json"
+    json_headers = {CONTENT_TYPE: APPLICATION_JSON}
+
+    def get_person_by_person_ids(self, person_ids, session_id=None, nonce=None, pretty_print=False):
+        request_model = GetPersonByPersonIDsRequest(person_ids=person_ids, session_id=self._get_session_id(session_id),
+            nonce=self._get_nonce(nonce), pretty_print=pretty_print)
+        return self.get(resource_endpoint=ApiEndpoints.GET_PERSON_BY_PERSON_IDS,
+            response_model=GetPersonByPersonResponse, params=request_model.as_params_dict)
```
Comment (darkhipenko) [inline APIs/persons/client.py:27]: GetPersonByPerson**IDs**Response

File: APIs/persons/client.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    def get_user_details(self, session_id=None, nonce=None, pretty_print=False):
+        request_model = GetUserDetailsRequest(session_id=self._get_session_id(session_id), nonce=self._get_nonce(nonce),
```
Comment (darkhipenko) [inline APIs/persons/client.py:30]: Base request, no need to create a new class for sending request

File: APIs/persons/requests/get_user_details.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+from base.common.models.request import BaseRequestModelKeys, SimpleRequestModel
+
+
+class GetUserDetailsRequest(SimpleRequestModel):
+    def __init__(self, session_id, nonce, pretty_print):
+        super().__init__(session_id=session_id, nonce=nonce, pretty_print=pretty_print)
+
+    def to_params(self):
+        args = super().to_params()
+        return args
```
Comment (darkhipenko) [inline APIs/persons/requests/get_user_details.py:11]: Extra code, use base request instead

**Review by darkhipenko — CHANGES_REQUESTED** [review]
A few changes are needed to be done

File: APIs/persons/client.py (hunk 1 above)
Comment (azayatsfv) [reply APIs/persons/client.py]: fixed

File: APIs/persons/requests/get_user_details.py (hunk 3 above)
Comment (azayatsfv) [reply APIs/persons/requests/get_user_details.py]: deleted

File: APIs/persons/client.py (hunk 2 above)
Comment (azayatsfv) [reply APIs/persons/client.py]: fixed

**Review by darkhipenko — APPROVED** [review]
Approved

**Pull request merged**